includes/discovery/sensors/fanspeeds/lmsensors.inc.php

```diff
@@ -19,7 +19,7 @@
             $oid              = '1.3.6.1.4.1.2021.13.16.3.1.3.'.$index;
             $current          = snmp_get($device, $oid, '-Oqv', 'LM-SENSORS-MIB');
             $descr            = trim(str_ireplace('fan-', '', $descr));
-            if ($current > '0') {
+            if ($current !== false && $current >= 0) {
```

Comment:
BTW, it doesn't matter that `false >= 0 == true`, because that means `false !== false` already failed the check.  So it is fine as stands.